### ISSUE TITLE
BIG CHANGE NEEDED

### ISSUE THREAD

**Issue by miuthe** [imyizhang/suno-api#6]
suno changed it's url from .ai to .com
so the code needs to be adjusted else you get a 'this is a fake app' waterkmark in everysong

**Comment by miuthe** [comment]
it's basically just changing to studio-api.prod.com (except for cdn1)

**Comment by oppenheimer-** [comment]
I managed to overcome these initial hurdles, but now there's an error (status 422 Unprocessable Content) when trying to generate a song.

ideas?

**Comment by miuthe** [comment]
yes, they added a really strong bot detector ig, you need to manually create a song then it'll work for a few hours before getting the same error 
same with token validation failed

**Comment by oppenheimer-** [comment]
just to make sure i got this: the cookie renews as well?

**Comment by miuthe** [comment]
i dont think so idk tho

**Comment by Lynn0990** [comment]
Exciting News! The Watermark-Free Version of SUNO Music API is Now Live! Fully Supports the Latest V4 Version, More Powerful Features!
(99% of APIs on the market may still have the "This is a fake app" watermark)

 Welcome to experience for free: https://surl.id/1uHIdUobVg
API: https://surl.id/1uHIdUmZQU
Documentation: https://surl.id/1uHIdUmyFr

Background:
Recently, SUNO introduced the robust hCaptcha security mechanism, resulting in nearly 99% of APIs on the market embedding the "This is a fake app" watermark in generated audio. This measure not only targets third-party cracking services but also accurately identifies genuine user verification behaviors. APIs that fail to completely bypass hCaptcha will include the "This is a fake app" tag in their output.Through relentless effort and in-depth research, our team has successfully circumvented this restriction using advanced high-fidelity captcha cracking simulation techniques. 
 
Important Reminder:
We will also launch new discounted prices in the near future, and welcome everyone to experience them！

**Comment by nqb992** [comment]
好消息！SUNO 音乐 API 无水印版本已经成功上线！（市面上 99% 的 API 可能都带有水印 This is a fake app）

API: "https://surl.id/1uHGqFVHSD"
对接文档: "https://surl.id/1uHGqFWkp4"
同时支持最新 V4 版本！

背景：SUNO 近期上线了 hCaptcha，导致几乎 99% 的 API 的生成都出现了问题，生成的音频里面带有 This is a fake app 内容。hCaptcha 一方面打击第三方破解服务，一方面对真实用户真实样子呢验证行为进行识别，如果不能全部破解，生成结果就会带有 This is a fake app。我放经过多方面研究探索，经过高仿真验证码破解模拟实现，我们成功绕过了此限制，市面上 99% 的服务可能都未解决此难题。欢迎大家测试使用～

注意：⚠ 目前我们的 SUNO API 几乎是全网最低价。但由于新型破解方案成本变高，本 API 短期即将涨价不少于 30%，目前暂时还是原价，需要的朋友尽快趁低价购买。




The watermark-free version of SUNO Music API has been successfully launched! Supports the latest V4! No This is a fake app! (99% of APIs on the market may have the watermark This is a fake app)

API: https://surl.id/1uHGqFVHSD
Documentation: https://surl.id/1uHGqFWkp4

Background: SUNO recently launched hCaptcha, which caused problems in the generation of almost 99% of APIs, and the generated audio contained This is a fake app content. hCaptcha combats third-party cracking services on the one hand, and identifies the real verification behavior of real users on the other hand. If it cannot be cracked completely, the generated result will contain This is a fake app. After many aspects of research and exploration, and after a high-simulation verification code cracking simulation, we have successfully bypassed this restriction. 99% of the services on the market may not have solved this problem. Welcome everyone to test and use it~
